src/Phoenix.vue

```diff
@@ -140,9 +176,18 @@ export default {
     }
     return metaInfo
   },
+
+  mounted() {
+    this.$nextTick(() => {
+      window.addEventListener('resize', this.onResize)
```

Comment:
note: we have a "resize-observer" plugin, but it's more designed for specific elements in the DOM, not the window. I guess you did not need it

Reply:
Yes, didn't see a reason to use it here since we are watching the whole window. Also, to try not to have an additional element in the DOM.